### PR TITLE
[KIECLOUD-225] - keeping permissions to old eap data-dir

### PR DESCRIPTION
**Pull request by ricardozanini**
See: https://issues.jboss.org/browse/KIECLOUD-225
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[KIECLOUD-XYZ] Subject`, `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: jboss-kie-workbench/configure.sh [hunk 1]
```diff
@@ -16,8 +16,12 @@ chown -R jboss:root ${JBOSS_HOME}/bin/
 chmod -R g+rwX ${JBOSS_HOME}/bin/
 
 # Ensure that the local data directory exists
-DATA_DIR=/opt/kie/data
+DATA_DIR=${JBOSS_HOME}/standalone/data
+KIE_HOME_DIR=/opt/kie
```
Comment (errantepiphany) [inline jboss-kie-workbench/configure.sh]: What about?
KIE_DATA_DIR=/opt/kie/data
... etc ...

Comment (errantepiphany) [reply jboss-kie-workbench/configure.sh]: JBOSS_DATA_DIR ?
KIE_DATA_DIR ?

Comment (ricardozanini) [reply jboss-kie-workbench/configure.sh]: This dir is manipulated by the PV. If in the future we create another one like /opt/kie/cert, the permissions will inherit.

Comment (errantepiphany) [reply jboss-kie-workbench/configure.sh]: sounds good.

**Pull request merged**